### ISSUE TITLE
Bug: Not able to Approve in case Email contains Upper-case letters. 

### ISSUE THREAD

**Issue by pawan-59** [devtron-labs/devtron#5092]
### 📜 Description

Recently, I faced an issue on the platform, my actual email ID is something like Test@example.com and we added a user as test@example.com hence, the user with approval access was not able to approve(configuration changes and images).

### Affected areas

RBAC Issues

### Additional affected areas

CD

### Prod/Non-prod environments?

Prod

### Is User unblocked?

No

### How was the user un-blocked?

None

### Impact on Enterprise

Users are not able to approve configuration changes and images.

### 👟 Steps to replicate the Issue

1. Add email on the platform in upper case(a few letters) that not should be in a small case.
2. Try to use protection configuration and image approver feature.

### 👍 Expected behavior

The user should be able to use the above feature.

### 👎 Actual Behavior

Not able to approve the configuration, approve button faded.

### ☸ Kubernetes version

1.27 EKS

### Cloud provider

<details>
AWS
</details>

### 🌍 Browser

Chrome

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find any similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/devtron-labs/devtron/blob/main/CODE_OF_CONDUCT.md)

AB#9775

**Comment by abhibhaw** [comment]
Should be fixed with https://github.com/devtron-labs/devtron/pull/5196 Duplicate of #5278
